colorpickerview/src/main/java/com/skydoves/colorpickerview/ColorPickerView.java

```diff
@@ -116,6 +116,26 @@ public ColorPickerView(Context context, AttributeSet attrs, int defStyleAttr) {
     onCreate();
   }
 
+  @Override
+  public boolean isEnabled() {
+    return super.isEnabled();
+  }
+
+  @Override
+  public void setEnabled(boolean enabled) {
+    super.setEnabled(enabled);
+
+    selector.setVisibility(enabled ? VISIBLE : INVISIBLE);
+    alphaSlideBar.setEnabled(enabled);
+    brightnessSlider.setEnabled(enabled);
```

Comment:
If we call the `setEnabled` before setting the sliders, we will get a `NullPointerException`.
Let's check the null safety like below.

```java
    if (getAlphaSlideBar() != null) {
      getAlphaSlideBar().setEnabled(enabled);
    }

    if (getBrightnessSlider() != null) {
      getBrightnessSlider().setEnabled(enabled);
    }
```